src/DynamoApplications/StartupUtils.cs

```diff
@@ -64,7 +67,7 @@ internal class CMDLineOptions
         public bool ServiceMode { get; set; }
     }
 
-    public class StartupUtils
+    public static class StartupUtils
```

Comment:
would it be too much of a breaking change to make this internal ?

Reply:
Why are you suggesting we make this internal?

Reply:
Just to reduce the number public footprint. Do you think it is valuable as public facing API or risky to make internal?